servers/src/main/java/tachyon/master/rawtable/RawTableMasterServiceHandler.java

```diff
@@ -20,13 +20,17 @@
 
 import tachyon.TachyonURI;
 import tachyon.exception.TachyonException;
+import tachyon.replay.ReplayCache;
 import tachyon.thrift.RawTableInfo;
 import tachyon.thrift.RawTableMasterService;
+import tachyon.thrift.RpcOptions;
 import tachyon.thrift.TachyonTException;
 import tachyon.thrift.ThriftIOException;
 
 public class RawTableMasterServiceHandler implements RawTableMasterService.Iface {
   private final RawTableMaster mRawTableMaster;
+  /** Replay cache used for RPCs which return values of type {@link Long}. */
+  private final ReplayCache<Long> mReplayCacheLong = ReplayCache.newInstance();
```

Comment:
How about we could cache `Object`s and downcast them before they are returned? That way we can have a single cache per master.